src/test/java/pkmncore/SearchTest.java

```diff
@@ -0,0 +1,16 @@
+package pkmncore;
+
+import com.google.gson.JsonObject;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class SearchTest {
+
+    @Test
+    public void getsAFakeJSONResponse() {
+        SearchFake fake = new SearchFake();
+        JsonObject response = fake.findByName("pika");
+        assertEquals("pikachu", response.get("name").getAsString());
+    }
+}
```

Comment:
Here it looks like you are testing the Fake - we don't usually do this, unless there is a lot of `important` logic in the test double.